packages/nextjs/app/builders/0x033ebeaBCC05eBaF5628b7bDDfacaDF1426efAA4/page.tsx

```diff
@@ -0,0 +1,86 @@
+//import { BellIcon, BriefcaseIcon, CodeBracketIcon } from "@heroicons/react/24/outline";
+import DynamicHeroIcon from "./DynamicHeroIcon";
+import type { NextPage } from "next";
+
+//import DynamicHeroIcon from "./DynamicHeroIcon";
+
+const Hugo: NextPage = () => {
+  return (
+    <>
+      <div className="text-center px-5 max-w-xl mx-auto pt-10">
+        <div className="px-5">
+          <h1 className="text-center">
+            <span className="block text-2xl mb-2">Hi Batch 6 Team!</span>
+            <span className="block text-4xl font-bold">I am Hugo</span>
+          </h1>
+          <p className="text-center text-lg mt-4">
+            I am an experienced SAP developer. Lastly playing with web programming and solidity. I am fascinated with
+            ethereum world and all the potentiality that smartcontracts can provide. I expect to learn a lot with
+            BuidlGuidl and be helpful for you all!
+          </p>
+        </div>
+      </div>
+      {/* </section> */}
+
+      <div className="flex-grow w-full mt-16 px-8 py-12 bg-base-300">
+        <nav className="flex justify-center items-center gap-8 flex-col sm:flex-row">
+          {[
+            ["Explore my projects on", "GitHub", "https://github.com/hdevelopergit", "CodeBracketIcon"],
+            [
+              "Connect with me on",
+              "Linkedin",
+              "https://www.linkedin.com/in/hugo-hernán-angarola-bb271a7a",
+              "BriefcaseIcon",
+            ],
+            ["Follow me on", "Twitter", "https://x.com/hugo33612165", "BellIcon"],
+          ].map(([text, title, url, icon]) => (
+            <div
+              key={title}
+              className="flex flex-col bg-base-100 px-6 py-6 text-center items-center max-w-xs rounded-3xl"
+            >
+              {/* <BellIcon className="h-8 w-8 fill-secondary"/> */}
+              <DynamicHeroIcon icon={icon} />
+              <p>
+                {text}{" "}
+                <a href={url} target="_blank" className="link">
+                  {title}
+                </a>
+              </p>
+            </div>
+          ))}
```

Comment:
Umm instead of inlining the array in jsx and using `DynamicHeroIcon` we could do something like: 

```tsx
import type { NextPage } from "next";
import { BellIcon, BriefcaseIcon, CodeBracketIcon } from "@heroicons/react/24/outline";

const myData = [
  ["Explore my projects on", "GitHub", "https://github.com/hdevelopergit", CodeBracketIcon],
  ["Connect with me on", "Linkedin", "https://www.linkedin.com/in/hugo-hernán-angarola-bb271a7a", BriefcaseIcon],
  ["Follow me on", "Twitter", "https://x.com/hugo33612165", BellIcon],
] as const;

const Hugo: NextPage = () => {
       return (
       {/* ... */}
        <nav className="flex justify-center items-center gap-8 flex-col sm:flex-row">
          {myData.map(([text, title, url, Icon]) => (
            <div
              key={title}
              className="flex flex-col bg-base-100 px-6 py-6 text-center items-center max-w-xs rounded-3xl"
            >
              {/* <BellIcon className="h-8 w-8 fill-secondary"/> */}
              <Icon className="h-8 w-8 fill-secondary" />
              <p>
                {text}{" "}
                <a href={url} target="_blank" className="link">
                  {title}
                </a>
              </p>
            </div>
          ))}
        </nav>
       {/* ... */}       
       )
 };
```

This way we don't need to use of hacky `DynamicHeroIcon` component and also easy for readability ?

Reply:
I will review. Honestly I'm not a web developer so for me all of this is new :)
The solution I've impelemented I had found googling. I will try to approach. Thanks!

Reply:
Hi @technophile-04 . I've implemented your approach and works perfectly. It's much more elegant this way. Thanks for suggestion! I've also deleted the comments in the coding. I hope now for the final approval 🙏😊